rust/src/core/polars/mod.rs

```diff
@@ -210,7 +228,18 @@ impl DPNamespace {
     /// # Arguments
     /// * `scale` - Noise scale parameter for the Laplace distribution. `scale` == standard_deviation / sqrt(2).
     pub fn laplace(self, scale: Option<f64>) -> Expr {
-        apply_anonymous_function(vec![self.0], LaplaceArgs { scale })
+        self.noise(scale, Some(Distribution::Laplace))
+    }
+
+    /// Add Gaussian noise to the expression.
+    ///
+    /// `scale` must not be negative or inf.
```

Comment:
If it is, what happens? Should it panic, or is the behavior undefined?

Reply:
You'll get an error at construction-time. If you try to evaluate it directly, you'll get a data-independent error at runtime.